application/common/sync/Synchronizer.php

```diff
@@ -205,6 +286,14 @@ protected function deactivateUsers($employeeIdsToDeactivate)
      */
     protected function getAllIdBrokerUsersByEmployeeId($fields = null)
     {
+        if ( ! in_array(User::EMPLOYEE_ID, $fields)) {
```

Comment:
Oh, good catch. We probably don't want an empty array, because that would end up telling `$this->idBroker->listUsers($fields);` (below) that we want no fields, rather than that we're not specifying (in which case it will give us a default set of fields).

However, you're right that I need to deal with the `null` before trying to use it in `in_array(...)`. Thanks!